osu.Game/Online/API/Requests/Responses/APIRelation.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using Newtonsoft.Json;
+
+namespace osu.Game.Online.API.Requests.Responses
+{
+    public class APIRelation
+    {
+        [JsonProperty("target_id")]
+        public int TargetID { get; set; }
+
+        [JsonProperty("relation_type")]
+        public RelationType RelationType { get; set; }
+
+        [JsonProperty("mutual")]
+        public bool Mutual { get; set; }
+
+        [JsonProperty("target")]
+        public APIUser? TargetUser { get; set; }
```

Comment:
Where is this field referenced from? It's not in https://github.com/nanaya/osu-web/blob/c029567dce89fd68446f56e40870c3cec01d7591/resources/js/interfaces/user-relation-json.ts#L4-L8

Reply:
But he does exist, I think it's this is in
https://github.com/nanaya/osu-web/blob/fe21e207e8f8106ae024d8eab9108b8481ea1b5a/app/Http/Controllers/FriendsController.php#L57

api will return APIUser when request friends but not for block relation